client/js/options.js

```diff
@@ -1,6 +1,7 @@
 "use strict";
 
 const $ = require("jquery");
+const _ = require("lodash");
```

Comment:
If requiring `lodash.escaperegexp` does not bring half of lodash with it, then that can work, otherwise just copy the function (it's a oneliner anyway).